sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -111,8 +111,10 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
     renderer.onStart()
     pluginRegistry.onStart()
     if (!mapboxMap.isStyleLoadInited) {
-      // Load the default style if no style has loaded yet.
-      mapboxMap.loadStyleUri(Style.MAPBOX_STREETS)
+      // Load the style in mapInitOptions if no style has loaded yet.
+      mapInitOptions.styleUri?.let {
+        mapboxMap.loadStyleUri(it)
+      }
```

Comment:
cc @julianrex to align the behaviour on iOS.

Reply:
also note that there's a ticket proposing to rename mapboxMap.iisStyleLoadInited https://github.com/mapbox/mapbox-maps-android-internal/issues/352